index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A form to allow CEAS tribunal members to submit reimbursement requests.">
```

Comment:
Capitilize "Tribunal":
```suggestion
    <meta name="description" content="A form to allow CEAS Tribunal members to submit reimbursement requests">
```